libr/arch/p/arm/armass64.c

```diff
@@ -1,5 +1,9 @@
 /* radare - LGPL - Copyright 2015-2023 - pancake */
 
+#include "r_types.h"
```

Comment:
define R_LOG_ORIGIN before any include

Reply:
Not sure I understand this one, I removed r_types.h but is there an alternative header I should be using instead?